### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by FannyGautierr**
Hey :wave:,

This PR will fix :

1 typo in `examples/Clustering.ipynb`
2 typos in `examples/Creating_slides_with_Assistants_API_and_DALL-E3.ipynb`
1 typo in `examples/Fine_tuning_for_function_calling.ipynb`
2 typos in `examples/How_to_call_functions_with_chat_models.ipynb`
2 typos in `examples/Question_answering_using_a_search_API.ipynb`
1 typo in `examples/Semantic_text_search_using_embeddings.ipynb`
3 typos in `examples/data/scifact_claims.jsonl`
1 typo in `examples/evaluation/Evaluate_RAG_with_LlamaIndex.ipynb`
2 typos in `examples/fine-tuned_qa/ft_retrieval_augmented_generation_qdrant.ipynb`
2 typos in `examples/third_party/GPT_finetuning_with_wandb.ipynb`
1 typo in `examples/vector_databases/cassandra_astradb/Philosophical_Quotes_CQL.ipynb`
1 typo in `examples/vector_databases/milvus/Getting_started_with_Milvus_and_OpenAI.ipynb`
1 typo in `examples/vector_databases/mongodb_atlas/semantic_search_using_mongodb_atlas_vector_search.ipynb`
1 typo in `registry.yaml`

Best!

### CODE REVIEW

**Comment by kesku** [comment]
Were these found manually or programmatically? I noticed for the `registry.yaml` typo there was a related pull request 3 months back - #798, which didn't fix the issue in the registry

**Comment by FannyGautierr** [comment]
Programmatically but they have been reviewed manually

**Review by Helfese — COMMENTED** [review]
Nice spotting

File: examples/Question_answering_using_a_search_API.ipynb [hunk 1]
```diff
@@ -398,7 +398,7 @@
      "text": [
       "Top 5 articles: \n",
       "\n",
-      "Title: NBA Finals: Denver Nuggets beat Miami Hea, lift thier first-ever NBA title\n",
+      "Title: NBA Finals: Denver Nuggets beat Miami Hea, lift their first-ever NBA title\n",
```
Comment (kesku) [inline examples/Question_answering_using_a_search_API.ipynb:401]: Should change `Miami Hea` to `Miami Heat` - the report from Business Standard weirdly has 2 spelling errors in the title

File: examples/Question_answering_using_a_search_API.ipynb [hunk 2]
```diff
@@ -461,7 +461,7 @@
...
+       "The Denver Nuggets won their first-ever NBA championship by defeating the Miami Heat 94-89 in Game 5 of the NBA Finals held on Tuesday at the Ball Arena in Denver, according to this [Business Standard article](https://www.business-standard.com/sports/other-sports-news/nba-finals-denver-nuggets-beat-miami-hea-lift-their-first-ever-nba-title-123061300285_1.html). Nikola Jokic, the Nuggets' center, was named the NBA Finals MVP. In a rock-fight of a Game 5, the Nuggets reached the NBA mountaintop, securing their franchise's first NBA championship and setting Nikola Jokic's legacy as an all-timer in stone, according to this [Yahoo Sports article](https://sports.yahoo.com/nba-finals-nikola-jokic-denver-nuggets-survive-miami-heat-to-secure-franchises-first-nba-championship-030321214.html). For more information and photos of the Nuggets' celebration, check out this [Al Jazeera article](https://www.aljazeera.com/gallery/2023/6/15/photos-denver-nuggets-celebrate-their-first-nba-title) and this [CNN article](https://www.cnn.com/2023/06/12/sport/denver-nuggets-nba-championship-spt-intl?cid=external-feeds_iluminar_yahoo)."
```
Comment (kesku) [inline examples/Question_answering_using_a_search_API.ipynb:464]:
```ipynb
[Business Standard article](https://www.business-standard.com/sports/other-sports-news/nba-finals-denver-nuggets-beat-miami-hea-lift-their-first-ever-nba-title-123061300285_1.html)
```

You've mentioned this was done programmatically and manually reviewed - I do find it odd the URL typo of `thier` was changed to `their` though. The typo exists in the article's title on their website and the original URL for the article page

**Review by kesku — COMMENTED** [review]
Looks good, though I noted 2 minor things

File: examples/Question_answering_using_a_search_API.ipynb (hunk 2 above)
Comment (FannyGautierr) [reply examples/Question_answering_using_a_search_API.ipynb:464]: Yeah sorry, careless mistake

**Comment by pierpaolo28** [comment]
There is also a typo [here (Moderation Guardrail section).](https://cookbook.openai.com/examples/how_to_use_guardrails#moderation-guardrail) 

"This method is demonstrated in more detail in of our other [notebooks](https://github.com/openai/openai-cookbook/blob/main/examples/evaluation/How_to_eval_abstractive_summarization.ipynb)." -> "This method is demonstrated in more detail in **one** of our other [notebooks](https://github.com/openai/openai-cookbook/blob/main/examples/evaluation/How_to_eval_abstractive_summarization.ipynb)."

**Pull request closed without merging**